omnibus/config/projects/agent.rb

```diff
@@ -157,6 +157,11 @@
 # Datadog agent
 dependency 'datadog-agent'
 
+# System-probe
+if linux?
+  dependency 'system-probe'
+end
+
```

Comment:
Why is this required now ?

Reply:
That’s because I removed the `invoke system-probe.build` invocation that there was in `omnibus/config/software/datadog-agent.rb` line 124.